api/src/main/java/org/pmiops/workbench/reporting/ReportingServiceImpl.java

```diff
@@ -60,6 +62,9 @@ public void collectRecordsAndUpload() {
     reportingQueryService
         .getWorkspacesStream()
         .forEach(b -> reportingUploadService.uploadBatchWorkspace(b, captureTimestamp));
+    reportingQueryService
+        .getUserStream()
+        .forEach(b -> reportingUploadService.uploadBatchWorkspace(b, captureTimestamp));
```

Comment:
hm.. does this compile? uploadBatch**Workspace**

Reply:
nope

Reply:
I mean - presumably this is a typo and should have been `uploadBatchUser` - why does CI pass though with this bug?

Reply:
It is compiled by CI, but that is a mistake. Sorry for the confusion, I let getUserStream returns Workspaces, and just realize we don't have test for `getUserStream()`
Added one for that.